src/IXmlLoader.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright 2018 ThoughtWorks, Inc.
+//
+// This file is part of Gauge-Dotnet.
+//
+// Gauge-Dotnet is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Gauge-Dotnet is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Gauge-Dotnet.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Collections.Generic;
+using System.Xml.Linq;
+
+namespace Gauge.Dotnet
+{
+    public interface IXmlLoader
```

Comment:
this interface is a specific contract, ex. get list of removed attributes. The interface name is too generic. Suggest renaming it to something more specific.